src/kernel_abi.h

```diff
@@ -2125,6 +2125,26 @@ struct ARM64Arch : public GenericArch<SupportedArch::aarch64, WordSize64Defs> {
   };
   typedef struct user_fpsimd_state user_fpregs_struct;
 
+  struct sigcontext {
+    __u64 fault_addr;
+    user_pt_regs regs;
+    // ISA extension state follows here
+  };
+
+  struct ucontext {
```

Comment:
Hmm why aren't you RR_VERIFY_TYPE_ARCHing any of these?

Reply:
rt_sigframe isn't in the uapi. struct sigcontext as defined in the kernel header is useless, because they just stuck a page size u8 array at the end, so taking sizeof of it doesn't give the correct size that the kernel actually stores.